order.html

```diff
@@ -0,0 +1,159 @@
+            <tr><th><center><a href="desserts.html" style="color: white;">Desserts</a><br></center></th></tr>
+            <tr><th><center><a href="bakery.html" style="color: white;">Bakery</a><br></center></th></tr>
+            <tr><th><center><a href="Others.html" style="color: white;">Others</a><br></center></th></tr>
+          </table>
+        </nav>
+        </div>
+
+<center>
+
+            <h1 style="color:black;font-size: 35px;background-color: #ffd603;border-radius: 15px;height: 40px;width: 30%;font-family: Lobster,cursive, sans-serif;
+            text-shadow: 1px 1px 0 #444f">Order Your Food</h1>
+        </center>
+        <div class="f">
+      <form style="color: white;" class="tab">
+        <table align="center" style="width :100%;" >
+          <tr >
+            <td>
+                <label for= "Name" >Full Name: </label>
+                </td><td>
+                <input type ="text" id="txtName" class ="ab" placeholder="FirstName LastName"><br/>
+                </td>
+              </tr><br>
+              <tr><td>
+                <label for= "email" >Email:</label>
+                </td><td>
+                <input type ="email" id="txtemail" class="ab" placeholder="EMAIL_ADDRESS"><br/>
+                </td></tr>
+                <tr><td>
+                <label for= "address" >Address:</label>
+                </td><td>
+                <input type ="text" id="address" class="ab" placeholder="Address"><br/>
+                </td></tr>
+                <tr><td>
+                <label for= "order">Your Order:</label>
+                </td><td>
+                    <select id ="tourpckg" > 
+                         <option>Red Rice & Curry</option>
+                         <option>Noodles</option> 
+                         <option>String Hoppers </option> 
+                         <option>Bread</option> 
+                         <option>parotta</option>
+                         <option>Omelette</option> 
+                         <option>Vegetable Salad</option> 
+                         <option> Chicken Soup </option> 
+                         <option>Normal Rice & Curry</option>
+                         <option>Rice & Curry (Fish)</option> 
+                         <option>Rice & Curry (Chicken)</option> 
+                         <option>Fried Rice</option> 
+                         <option>Chicken Fried Rice</option>
+                         <option>Omlet Fried Rice</option> 
+                         <option>Mixed Rice</option> 
+                         <option> Nasiguran</option> 
+                         <option>sandwich</option> 
+                         <option>pasta</option> 
+                         <option> rolls </option>
+                         <option>burger</option> 
+                         <option>Sausage Rolls</option> 
+                         <option> Pastries </option>  
```

Comment:
You have to populate these items  from the database that's why I asked you in my review comment to make files .php extension